Can't connect to the database :(

Hi, guys, 

I'm getting this: "error establishing a database connection" and can't debug it and connect the DB with the .env file. Is there something basic I'm missing? I'm not very familiar with docker and Symfony 6, I'm working with Symfony 4 mostly. 

Regards, 
Boris

I'm also having this issue. Appears the mysql container is failing to initialise due to the following error:

```
mysql_1    | 2024-05-02T14:59:48.142891Z 0 [ERROR] [MY-000067] [Server] unknown variable 'default-authentication-plugin=mysql_native_password'.
mysql_1    | 2024-05-02T14:59:48.143475Z 0 [ERROR] [MY-010119] [Server] Aborting
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -175,7 +175,7 @@ jobs:
                     docker run --rm -t -e HOST_PWD="$PWD" \
                         -v "$PWD":/app -v /var/run/docker.sock:/var/run/docker.sock \
                         projectinstaller testproject -v
-                    sleep 20
+                    sleep 30
 
             -   name: Check that website is up and running
                 run: |

--- a/install/docker-compose.prod.yml
+++ b/install/docker-compose.prod.yml
@@ -44,8 +44,7 @@ services:
             - traefik.http.services.${APP_NAME:-sword}.loadbalancer.server.port=80
 
     mysql:
-        image: mysql:8
-        command: --default-authentication-plugin=mysql_native_password
+        image: mysql:8.0.37
         restart: 'no'
         environment:
             - MYSQL_USER=ChangeMe
@@ -56,7 +55,7 @@ services:
             - mysql_data:/var/lib/mysql
 
     traefik:
-        image: traefik:2.9
+        image: traefik:2.11
         ports:
             - '443:443'
         volumes:

--- a/install/docker-compose.yml
+++ b/install/docker-compose.yml
@@ -53,8 +53,7 @@ services:
             - traefik.http.services.${APP_NAME:-sword}.loadbalancer.server.port=80
 
     mysql:
-        image: mysql:8
-        command: --default-authentication-plugin=mysql_native_password
+        image: mysql:8.0.37
         restart: 'no'
         environment:
             - MYSQL_USER=user
@@ -83,7 +82,7 @@ services:
             - traefik.tcp.services.${APP_NAME:-sword}-smtp.loadbalancer.server.port=1025
 
     traefik:
-        image: traefik:2.9
+        image: traefik:2.11
         ports:
             - '443:443'
         volumes:

